src/ai/backend/manager/api/exceptions.py

```diff
@@ -256,6 +256,11 @@ class VFolderCreationFailed(BackendError, web.HTTPBadRequest):
     error_title = 'Virtual folder creation has failed.'
 
 
+class VFolderDeletionFailed(BackendError, web.HTTPBadRequest):
```

Comment:
<img width="810" alt="image" src="https://user-images.githubusercontent.com/555156/175449657-dbe05515-10da-41a2-b5ba-b820af7ac94b.png">
Let's subclass `web.HTTPNotFound` and rename to `TooManyVFoldersFound` to keep consistency.

Reply:
The rationale behind subclassing "not found" for "too many" exceptions is that the precondition for "only one match wanted" is not satisfied and this is not a kind of user's fault nor system error.